lib/season.rb

```diff
@@ -84,9 +71,10 @@ def generate_accuracy_data
     team_games = @game_teams.group_by { |game_team| game_team.team_id }
     team_accuracy = {}
     team_games.each do |team_id, games|
-      accuracy_by_game = games.map { |game| game.accuracy }
```

Comment:
So this is where the discrepancy was?

Reply:
Yup. Turns out that taking an accuracy of each individual game, and then making an average out of all those accuracies, gives you sliiiightly different numbers than counting all goals for the season and dividing all shots for the season to get one main accuracy percentage.